assets/templates/dataset-filter/age-selector-list.tmpl

```diff
@@ -0,0 +1,73 @@
+								  <div class="panel">
+								  	<h3 class="margin-bottom--0 w-700">Some title</h3>
+								    <p class="margin-top--0">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Mauris vitae rhoncus diam. Vivamus non massa libero. Nulla non maximus sem. Fusce id vestibulum libero.</p>
+								  </div>
+								</details>
+							</div>
+							<form class="form adjust-font-size--16 clear-left" action="/jobs/{{.JobID}}/dimensions">
+								<div class="col col--md-29 col--lg-29">
+									<div class="checkbox margin-bottom--4">
+										<input type="checkbox" class="checkbox__input" id="All ages" name="All ages" value="All ages" checked>
+										<label class="checkbox__label" for="All ages">
+											All ages
+										</label>
+									</div>
+									<div class="margin-bottom--8">
+										<span class="block margin-bottom--2"><span class="font-size--17">Add from list  or</span> <a href="{{.Data.AddFromRange.URL}}">add age range</a> </span>
+
+										<div role="toolbar" aria-label="checkbox selector" class="margin-bottom--2 js--select-all-none nojs--hide">
+											<button type="button" id="select-all" aria-disabled="false" role="button" class="margin-right btn btn--link btn--link-underlined padding-top--0 padding-bottom--0">Select all</button>
+											<button type="button" id="select-none" aria-disabled="true" disabled role="button" class="margin-right btn btn--link btn--link-underlined btn--link-disabled padding-top--0 padding-bottom--0">Deselect all</button>
+										</div>
+										<div class="checkbox-group">
+											{{ $val := .Data.FiltersAmount }}
+											{{ range $i, $v := loop $.Data.AgeRange.StartNum $.Data.AgeRange.EndNum }}
+											<div class="checkbox">
+												<input type="checkbox" class="checkbox__input" id="id-{{$v}}" name="{{$v}}" value="{{$v}}">
+												<label class="checkbox__label" for="id-{{$v}}">
+													{{$v}}
+												</label>
+											</div>
+											{{end}}
+										</div>
+										<div class="margin-bottom--8">
+					              <input type="submit" value="Save and return to filter options" class="btn btn--primary btn--thick btn--wide btn--big btn--focus margin-right--2 font-weight-700 font-size--17 text-wrap">
+					              <a class="inline-block margin-top--4 font-size--17" href="{{.Data.Cancel.URL}}">Cancel</a>
+											</div>
+										</div>
+									</div>
+									<aside>
+										<div class="col col--md-14 col--lg-20 margin-left-md--4 margin-bottom--6 margin-left-lg--10">
+										  <div class="margin-bottom--6 hide--sm">
+										      <input type="submit" value="Save and return to filter options" class="btn btn--primary btn--thick btn--wide btn--big btn--focus font-weight-700 font-size--17 text-wrap full-width">
+										  </div>
+											{{ template "partials/filter-selection" . }}
+										</div>
+								</aside>
+							</form>
+          	</div>
```

Comment:
The formatting is off quite a bit in this file, might be worth sticking it through an HTML linter